dist.py

```diff
@@ -456,7 +457,11 @@ def build_linux(
 
             # Copy source tree to working directory.
             log(f'Copying source tree from: {source}')
-            shutil.copytree(source, f'{workdir}/cupy', symlinks=True)
+            dst = f'{workdir}/cupy'
+            shutil.copytree(source, dst, symlinks=True)
+
+            # Rename project name
+            rename_project(f'{dst}/pyproject.toml', package_name)
```

Comment:
nitpick: I'd want to avoid introducing new variables with ambiguous name here - it's already full of long-lived variables 😅

```suggestion
            shutil.copytree(source, f'{workdir}/cupy', symlinks=True)

            # Rename project name
            rename_project(f'{workdir}/cupy/pyproject.toml', package_name)
```